python/pylibcudf/pylibcudf/table.pyx

```diff
@@ -100,8 +121,10 @@ cdef class Table:
 
             tmp = Table.from_table_view_of_arbitrary(result.tbl.get().view(), result)
             self._columns = tmp.columns()
+        elif hasattr(arrow_like, "__arrow_c_device_stream__"):
```

Comment:
Once this is supported, would we make sure it's hit before the `elif hasattr(arrow_like, "__arrow_c_stream__"):` case?

Reply:
Good question. I think so, yes. I think we always want to prioritize the device protocols over the host protocols if both are defined (although we may also need to put some error checking in device protocols depending on what device the data is coming from), but for now we have to order it this way since this is an error case. I can add a note.